### DISCUSSION TITLE
在使用subjective数据集时报错

### DISCUSSION THREAD

**Discussion by hrdxwandg** [open-compass/opencompass#1459] [Q&A]
python run.py \
    --models hf_qwen2_1_5b_instruct \
    -datasets demo_gsm8k_chat_gen demo_math_chat_gen alignbench_judgeby_autoj \
    --debug \
    -a vllm

我在跑数据集alignbench_judgeby_autoj或者mtbench时，在eval阶段报错TypeError: LMEvaluator.__init__() missing 2 required positional arguments: 'judge_cfg' and 'output_path'。请问这个是什么原因，该怎么修改？

**Comment by bittersweet1999** [comment]
客观和主观的评测方式不一样，不能放在一起跑的，主观推荐用config的方式跑

**Reply by hrdxwandg** [reply]
> 客观和主观的评测方式不一样，不能放在一起跑的，主观推荐用config的方式跑

好的，明白了，多谢
